pkg/kubectl/serviceaccount.go

```diff
@@ -0,0 +1,79 @@
+/*
+Copyright 2016 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package kubectl
+
+import (
+	"fmt"
+
+	"k8s.io/kubernetes/pkg/api"
+	"k8s.io/kubernetes/pkg/runtime"
+)
+
+// ServiceAccountGeneratorV1 supports stable generation of a service account
+type ServiceAccountGeneratorV1 struct {
+	// Name of service account
+	Name string
+}
+
+// Ensure it supports the generator pattern that uses parameter injection
+var _ Generator = &ServiceAccountGeneratorV1{}
```

Comment:
> @deads2k I think I agree. People who want to pin to a specific behavior can use --api-version, right?

Our hubbed conversions allow us to set fields in one spot and translate it without losing its intent across versions.  The claim of a generator in the case `kubectl create <subcommand>` is that we've made a breaking API change in the case of simple construction.  It doesn't seem likely, but even if were the case I'd be more inclined to use a delegation pattern to actually use the old cli code itself.